benchmarks/test.nim

```diff
@@ -2,12 +2,8 @@ import zero_functional, strutils, sequtils, bench, times, os
 
 var data = readFile("data.txt")
 
-var a = data.split(" ").mapIt(parseInt(it))
-
-var b: seq[int] = @[]
-for z in 0..<2_000_000:
-  b.add(z)
-
+let a = data.split(" ") --> map(parseInt(it)) --> to(seq[int])
+let b = (0..<2_000_000) --> to(seq[int])
 
```

Comment:
This is so much faster than the original code - this itself is maybe already a great benchmark.
I guess - because `data.split()` and `0..2_000_000` are both iterators and because zero-functional directly works on it - instead of creating interim-lists maybe - I don't know, but it feels like running the test on my computer in less than 200msec instead of 2 seconds!

Reply:
hahaha I never used zero_functional for its own benchmark setup :D excellent